docs/release-notes/experiment-deadlock.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  API: Fix an issue where calling ``det job update`` could cause jobs to be prevented from
+   scheduling and ``det job ls`` to hang.
```

Comment:
:orphan:

**Bug Fixes**

-  API: Fix a bug where calling ``det job update`` could prevent jobs from being scheduled and ``det job ls`` to hang.